ajax/libs/jquery-nearest/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "jquery-nearest",
+    "description": "Find the elements in a page that are closest to a particular point or element, based on pixel dimensions",
+    "filename": "jquery.nearest.min.js",
+    "keywords": [
+        "jquery-plugin", 
+        "ecosystem:jquery", 
+        "nearest", "furthest", 
+        "touching", 
+        "pixel", 
+        "distance"
+    ],
+    "author": "Gilmore Davidson",
+    "homepage": "https://gilmoreorless.github.io/jquery-nearest/",
+    "repository": {
+        "type": "git",      
+        "url": "git+https://github.com/gilmoreorless/jquery-nearest.git"
+    },
+    "npmName": "jquery-nearest",
+    "npmFileMap": [
+    {
+      "basePath": "src",
+      "files": [
+        "*.*"        
```

Comment:
Let's use the pattern `**/jquery.nearest*`
Because the basePath is `src`, we just need to add the files which are `jquery.nearest*`
`files` is a pattern matcher allowing selection of multiple files, here is  [node-glob](https://github.com/isaacs/node-glob#glob-primer)